waspc/src/NpmDependency.hs

```diff
@@ -15,6 +17,9 @@ data NpmDependency = NpmDependency
 fromList :: [(String, String)] -> [NpmDependency]
 fromList = map (\(name, version) -> NpmDependency {_name = name, _version = version})
 
+printDep :: NpmDependency -> String
```

Comment:
"Philosophical" question here is if this function should be in `src/` or in `cli/`.
My first instinct is to put it in `cli/` since it is used there and there is no one right way to print an npm dependency, especially when you start making decisions like coloring - it really depends on where it is shown.
If there was one right way to print it, I would put it in `src/`, but since there isn't, it is tempting to put it in `cli/`.

On the other hand, how likely is it that we will need printing of dependency in `src/`? Maybe if there is some error and it is important to mention the dependency in that error.

Ok, I have to admit I am not sure in this case, if I really had to make a choice I would put it into `cli/`, but I am also ok with leaving it in `src/`.

Reply:
So directly into /cli/Command/Deps.hs?

Reply:
Hm, yes I think that would make sense for now.
But as I said, you don't have to do it, it is not really clear what is the best solution - up to you.